code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -969,7 +969,7 @@
 	else if(O.isscrewdriver())
 		if(length(contents) == 0)
 			to_chat(user, SPAN_NOTICE("You begin poking holes in \the [src]."))
-			if (do_after(user, 10/O.toolspeed, act_target = src))
+			if(O.use_tool(src, user, 30))
```

Comment:
is the removal of the toolspeed division in all of these intentional? also, what's up with increasing the timers? document them

Reply:
Yes. It's intentional. All toolspeed is handled by the use_tool proc itself, so people don't need to copypasta it everytime.

Reply:
ill add whatever timers i tweaked to the changelog